File: src/main/java/edu/put_the_machine/scrapper/model/parser_dto/UniversityLessons.java
package edu.put_the_machine.scrapper.model.parser_dto;

import lombok.EqualsAndHashCode;
import lombok.Getter;
import lombok.RequiredArgsConstructor;
import lombok.ToString;

import java.util.List;

@RequiredArgsConstructor
@Getter
@EqualsAndHashCode
@ToString
public class UniversityLessons {
    private final String universityName;
    private final List<GroupLessons> groupsLessons;
}

File: src/main/kotlin/edu/put_the_machine/scrapper/model/dto/parser/dto/UniversityLessons.kt
package edu.put_the_machine.scrapper.model.dto.parser.dto

import lombok.EqualsAndHashCode
import lombok.Getter
import lombok.RequiredArgsConstructor
import lombok.ToString

class UniversityLessons(
    val universityName: String,
    val groupsLessons: List<GroupLessons>
)